modules/acquia_cms_article/acquia_cms_article.install

```diff
@@ -19,3 +19,12 @@ function acquia_cms_article_install() {
     'delete any article content',
   ]);
 }
+
+/**
+ * Implements hook_update_N().
+ */
+function acquia_cms_article_update_8001() {
+  if (\Drupal::moduleHandler()->moduleExists('acquia_cms_common') && \Drupal::moduleHandler()->moduleExists('acquia_cms_site_studio')) {
+    _acquia_cms_common_update_view_display_options_style('article_cards', 'default', 'view_tpl_article_cards_slider');
+  }
```

Comment:
Also If I have to update this I would simply import the configuration again  and rest will be taken care by the eventSubscriber, this hook should be implemented as part of acqua_cms_article module itself instead of common let me know if you have any queries.

Reply:
Done @chandan-singh7929 
The Update hook is already in acquia_cms_article but updated the code to save view file programatically in update hook.